src/TaglibSharp/Tag.cs

```diff
@@ -1563,6 +1563,9 @@ public virtual void CopyTo (Tag target, bool overwrite)
 
 			if (overwrite || target.DateTagged == null)
 				target.DateTagged = DateTagged;
+
+			if (overwrite || target.Pictures.Length == 0)
```

Comment:
Please add null-safety: `if (overwrite || target.Pictures == null || target.Pictures.Length == 0)`

Reply:
You got it.